frontend/src/Components/Auth.js

```diff
@@ -1,4 +1,25 @@
 import {PostData} from "../serviceCalls"; 
+
+export function getVotes(callback) {
```

Comment:
I would add the "async" "await" things to this call since it is a post call and we don't want the program to run ahead without getting all the db info

Reply:
Oh jk I see a callback so we're fine

Reply:
I couldn't figure out how to get an async call in the componentDidMount function. So this was my fallback